.github/workflows/build.yml

```diff
@@ -90,33 +90,32 @@ jobs:
           - ubuntu-latest
           - windows-latest
         python:
-          - "3.9"
-          - "3.10"
-          - "3.11"
-          - "3.12"
           - "3.13"
-          - "3.14"
```

Comment:
These versions of Python helps us confirm that `aws-lambda-builders` **as a library** works in different versions of python (Meaning that if you have python3.9, you can do `pip install aws-lambda-builders` and you'll be able to use it correctly. 

Because this usage is limited (mostly used through SAM CLI anyway), then this is not the most crucial thing to check. But if it's not super slow or causing particular issues, we could probably add it back in the future.